File: app/src/main/java/com/perflyst/twire/model/Panel.java
package com.perflyst.twire.model;

/**
 * Created by Sebastian Rask on 24-02-2017.
 */

public class Panel {
    private final String mStreamerName;
    private final int mUserId;
    private final String mDescription;
    private final String mImageUrl;
    private final String mLinkUrl;
    private final String mTitle;
    private final String mHtml;
    private int mOrder;

    public Panel(String mStreamerName, int mUserId, int mOrder, String mDescription,
                 String mImageUrl, String mLinkUrl, String mTitle, String mHtml) {
        this.mStreamerName = mStreamerName;
        this.mUserId = mUserId;
        this.mOrder = mOrder;
        this.mDescription = mDescription;
        this.mImageUrl = mImageUrl;
        this.mLinkUrl = mLinkUrl;
        this.mTitle = mTitle;
        this.mHtml = mHtml;
    }

    public String getmHtml() {
        return mHtml;
    }

    public String getmStreamerName() {
        return mStreamerName;
    }

    public int getmUserId() {
        return mUserId;
    }

    public int getmOrder() {
        return mOrder;
    }

    public void setmOrder(int mOrder) {
        this.mOrder = mOrder;
    }

    public String getmDescription() {
        return mDescription;
    }

    public String getmImageUrl() {
        return mImageUrl;
    }

    public String getmLinkUrl() {
        return mLinkUrl;
    }

    public String getmTitle() {
        return mTitle;
    }
}

File: app/src/main/java/com/perflyst/twire/model/Panel.kt
package com.perflyst.twire.model

/**
 * Created by Sebastian Rask on 24-02-2017.
 */
class Panel(
    private val mStreamerName: String?,
    private val mUserId: String?,
    private var mOrder: Int,
    private val mDescription: String?,
    private val mImageUrl: String?,
    private val mLinkUrl: String?,
    private val mTitle: String?,
    private val mHtml: String
) {
    fun getmHtml(): String {
        return mHtml
    }

    fun getmStreamerName(): String? {
        return mStreamerName
    }

    fun getmUserId(): String? {
        return mUserId
    }

    fun getmOrder(): Int {
        return mOrder
    }

    fun setmOrder(mOrder: Int) {
        this.mOrder = mOrder
    }

    fun getmDescription(): String? {
        return mDescription
    }

    fun getmImageUrl(): String? {
        return mImageUrl
    }

    fun getmLinkUrl(): String? {
        return mLinkUrl
    }

    fun getmTitle(): String? {
        return mTitle
    }
}
